diff --git a/include/configs/zynq-common.h b/include/configs/zynq-common.h
index 39035f8b..c813573a 100644
--- a/include/configs/zynq-common.h
+++ b/include/configs/zynq-common.h
@@ -220,7 +220,7 @@
 
 
 /* Extend size of kernel image for uncompression */
-#define CONFIG_SYS_BOOTM_LEN	(60 * 1024 * 1024)
+#define CONFIG_SYS_BOOTM_LEN	(300 * 1024 * 1024)
 
 /* Boot FreeBSD/vxWorks from an ELF image */
 #define CONFIG_SYS_MMC_MAX_DEVICE	1
